helm/wasp-simple-h264-payload-parser/ci/ct-values.yaml

```diff
@@ -0,0 +1,30 @@
+config:
+  init:
+    registerThingType: true
+    thingServiceName: waspthingmock
```

Comment:
Seen a couple references like this, do we need to change these names?

Reply:
no, this is specifically for mocking the thing service when running `helm-test` and such